### DISCUSSION TITLE
Collecting latitude and longitude from a map for further use

### DISCUSSION THREAD

**Discussion by Mithra001** [apache/airflow#21449] [Q&A]
How can I use a geographical map to capture latitude and longitude data offline? I need to draw a square in the map and gather the latitude and longitude of each corner for later usage, and then show the latitude and longitude as a little dialogue box when the user isn't making a square (just moving across the map). I'm not sure how I'm going to accomplish it.

I'm hoping someone can assist me in resolving this problem!!

**Comment by potiuk** [comment]
This is quite likely not the right place to ask this question.
